src/core/ready.js

```diff
@@ -11,7 +12,9 @@ var readyList = jQuery.Deferred();
 
 jQuery.fn.ready = function( fn ) {
 
-	readyList.then( fn );
+	readyList
+		.then( fn )
+		.catch( jQuery.readyException );
```

Comment:
This should wrap `jQuery.readyException` so the lookup happens at the time of error handling rather than callback registration: `.catch( function( err ) { jQuery.readyException( err ); } )`.